lucien-ios/Models/AddEditComic.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  AddEditComic.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/20/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)